templates/repo/view_file.tmpl

```diff
@@ -137,4 +140,8 @@ function submitDeleteForm() {
 		$("#delete-file-form").submit()
 	}
 }
+function copy(link) {
```

Comment:
the name of `copy` is too short which may leads to conflict, maybe "copyLinkHref" is better and clear.

Reply:
Changed in https://github.com/go-gitea/gitea/pull/17145/commits/58a6bc9dd2f6a5e68b33c53d58561386f42512a9